docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       -P, --publish-all=false       Publish all exposed ports to random ports
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
+      --pids-limit=0                Tune container pids limit (set -1 for unlimited)
```

Comment:
I have been running this patch on several of our servers including prod ones to collect stats I'd be willing to show you the results